examples/advanced/example_model_optimization.py

```diff
@@ -277,16 +279,24 @@ def enc_to_repr(self, enc: torch.Tensor) -> torch.Tensor:
 #   If you do not specify an ``optimizer``, the
 #   :func:`~pystiche.optim.default_model_optimizer`, i.e. :class:`~torch.optim.Adam` is
 #   used.
+#
+# .. note::
```

Comment:
We should keep this note above the block that actually calls the training since at this point the pretrained weights are not yet introduced.